src/DynamoCoreWpf/UI/Converters.cs

```diff
@@ -3155,4 +3184,35 @@ public object ConvertBack(object value, Type targetType, object parameter, Cultu
             throw new NotImplementedException();
         }
     }
+
+    /// <summary>
+    /// Converts an ICollection<AnnotationViewModel> to a string
+    /// that displays how many AnnotationViewModels there is in the
+    /// Collection.
+    /// </summary>
+    [ValueConversion(typeof(ICollection<AnnotationViewModel>), typeof(string))]
+    public class NestedGroupsLabelConverter : IValueConverter
+    {
+        public object Convert(object value, Type targetType, object parameter, CultureInfo culture)
+        {
+            if (!(value is ICollection<AnnotationViewModel> viewModels) ||
+                !viewModels.Any())
+            {
+                return string.Empty;
+            }
+
+            var numberOfNestedGroups = viewModels.Count;
+            if (numberOfNestedGroups > 1)
+            {
+                return $"{numberOfNestedGroups} Groups";
```

Comment:
I think given we do not localize nodes and groups, this might be OK to stay as hard coded string, will take a note about this